.github/DISCUSSION_TEMPLATE/new-type.yml

```
@@ -1,59 +1,49 @@
 labels: ["New type"]
 body:
-- type: input
-  id: name
+- type: markdown
   attributes:
-    label: Type name
-    description: The name of the terms type to add, in Title Case.
-    placeholder: Business Privacy Policy
-  validations:
-    required: true
+    value: | 
+      Please share the details below to help the community to better understand your terms type suggestion.
 - type: textarea
   id: alternative-type-names
   attributes:
-    label: Alternative type names
+    label: What are the alternative names for this type?
     description: Known synonyms for the same terms type.
     placeholder: | 
       Business Users Privacy Policy
       Commercial Privacy Policy
-- type: input
-  id: writer
-  attributes:
-    label: Writer
-    description: The author of the document, in most cases the service provider itself.
-    placeholder: service provider
 - type: input
   id: audience
   attributes:
-    label: Audience
+    label: Who is the target audience for these terms?
     description: The targeted audience whose rights and duties are defined in the associated terms.
     placeholder: business user
 - type: input
   id: object
   attributes:
-    label: Object
-    description: The object of the commitment, i.e. the information or interaction whose handling will be constrained by the associated terms.
+    label: What is the object of commitment in these terms?
+    description: i.e. the information or interaction whose handling will be constrained by the associated terms.
     placeholder: personal data of business users
 - type: textarea
   id: examples
   attributes:
-    label: Examples
+    label: Can you provide examples of documents containing these terms?
     description: |
-      Examples of documents that contain terms of the suggested type, in the form `URL - Title - Service name`, one per line. Provide the URL address at which the document can be accessed; the title of the document provided at this URL; and the name of the service.
+      Examples of documents that contain terms of the suggested type, in the form `URL - Title - Jurisdiction`, one per line. Provide the URL address at which the document can be accessed; the title of the document provided at this URL; and, if possible, the jurisdiction to which the document refers.
     placeholder: | 
-      https://www.partners.skyscanner.net/privacy-policy - Corporate Business Contact Privacy Notice - Skyscanner
+      https://www.whatsapp.com/legal/privacy-policy-eea?lang=en - Privacy Policy - European Economic Area
   validations:
     required: true
 - type: textarea
   id: references
   attributes:
-    label: References
+    label: Are there any relevant references for this type?
     description: | 
-      Related resources that may help to understand the purpose of this type, such as legal definitions, or the discussions that led to the choice of this name.
+      Related references such as legal definitions, or the discussions that led to the choice of this name.
```

Let's try to use different words to increase the chances that at least some are clear to the reader 🙂 
```suggestion
      Related resources such as legal definitions, or the discussions that led to the choice of this name.
```